app/api-fetch/all-albums.ts

```
@@ -20,7 +18,7 @@
     try {
         if (localStorage.getItem("albums") !== null || Date.now() - parseInt(localStorage.getItem("albumsTime")!) < 3600000) {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)